app/models/event_streamer.rb

```diff
@@ -81,6 +83,7 @@ def start_heartbeat!
           sleep(5) # Timeout of 5 seconds
         end
       rescue IOError
+      rescue ActionController::Live::ClientDisconnected
```

Comment:
Rescues don't fall-through like this, so combine the lines like in my comment above.